tests/plugins/test_koji_import.py

```diff
@@ -239,12 +241,9 @@ def mock_environment(tmpdir, session=None, name=None,
     flexmock(subprocess, Popen=fake_Popen)
     flexmock(koji, ClientSession=lambda hub, opts: session)
     flexmock(GitSource)
-    if logs_return_bytes:
-        logs = b'build logs - \xe2\x80\x98 \xe2\x80\x97 \xe2\x80\x99'
-    else:
-        logs = 'build logs - \u2018 \u2017 \u2019'
+    logs = [LogEntry('x86_64', 'Hurray for bacon')]
```

Comment:
Maybe include a Unicode non-ASCII  code point like \u2017 to catch encoding issues.

Reply:
I'm starting to think that automatic UTF-8 encoding for files only works properly in Python 3. I can't seem to get it to work right in Python2.

So I think we should use an explicitly Unicode log message like `u'Hurray for bacon \u2017'` and fix up the `.write()` call so that it encodes the log line to 'utf-8' explicitly.